ykllvmwrap/src/ykllvmwrap.cc

```diff
@@ -72,6 +72,29 @@ std::unique_ptr<Module> load_module(LLVMContext &Context, char *Ptr,
   return M;
 }
 
+std::vector<Value*> get_trace_inputs(Function *F, uintptr_t BBIdx) {
+  std::vector<Value*> Vec;
+  auto It = F->begin();
+  // Skip to the first block in the trace which contains the `start_tracing`
+  // call.
+  std::advance(It, BBIdx);
+  BasicBlock *BB = &*It;
+  for (auto I = BB->begin(); I != BB->end(); I++) {
+    if (isa<CallInst>(I)) {
+      CallInst *CI = cast<CallInst>(&*I);
+      if (CI->getCalledFunction()->getName() == YKTRACE_START) {
+          auto Arg = CI->arg_begin();
+          // Skip first argument to start_tracing.
+          Arg++;
+          for (; Arg != CI->arg_end(); Arg++) {
```

Comment:
Would this be simpler as:

```c++
for (auto Arg = CI->arg_begin() + 1; ...)
```

?

Reply:
I think I initially tried `for (auto Arg = CI->arg_begin()++, ...` which didn't work. Your suggestion works though.
Fixed in 7b5d9d9.